src/FunctionalExtras/Data/TupleExtensions.cs

```diff
@@ -0,0 +1,34 @@
+using System;
```

Comment:
I think this file is misnamed.  It's dealing with `System.ValueTuple` instances, not `System.Tuple` instances.